.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: node_js
+sudo: false
+node_js:
+  - 6
+os:
+  - linux
+  - osx
+before_install:
+  - pip install --user cpplint
```

Comment:
Using `--user` here seems to break the builds on Mac OS, as it installs into a user's local base, which is probably not in the `$PATH` on the Mac OS machines...